values.yaml

```diff
@@ -51,6 +51,20 @@ coderd:
     # coderd.tls.devurlsHostSecretName -- The secret to use for DevURL TLS.
     devurlsHostSecretName: ""
 
+  clientTls:
+    # coderd.clientTls.certSecret -- Secret containing a PEM encoded cert file.
+    certSecret:
+      # coderd.clientTls.certSecret.name -- Name of the secret.
+      name: ""
+      # coderd.clientTls.certSecret.key -- Key pointing to a certificate in the secret.
+      key: ""
+    # coderd.clientTls.keySecret -- Secret containing a PEM encoded key file.
+    keySecret:
+      # coderd.clientTls.keytSecret.name -- Name of the secret.
+      name: ""
+      # .keySecret.key -- Key pointing to a certificate in the secret.
```

Comment:
should be coderd.clientTls.keySecret.key

Reply:
it's now `coderd.clientTLS.secretName`.